### PR TITLE
Hash WdlValues

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

**Comment by Horneth** [comment]
New PR to be merged to `job_avoidance` branch this time. I can squash this if it's :+1:

File: src/main/scala/cromwell/binding/values/WdlFile.scala [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package cromwell.binding.values
 
 import cromwell.binding.types.{WdlFileType, WdlType}
+import cromwell.engine.{FileHasher, Hash}
```
Comment (scottfrazer) [inline src/main/scala/cromwell/binding/values/WdlFile.scala]: Can we move Hash and FileHasher into the Binding layer?  The binding layer should not depend at all on the engine layer

Comment (Horneth) [inline src/main/scala/cromwell/binding/values/WdlFile.scala]: Good point, I wasn't sure where to put this in the first place

**Comment by scottfrazer** [comment]
:+1:

**Comment by cjllanwarne** [comment]
:+1:

**Pull request merged**